### PR TITLE
Disable import wallet button when the backup phrase is not valid

### LINKED ISSUE #802: Users SNBAT try to restore invalid backup phrases
### Expected Behavior

"Restore Celo Wallet" button is disabled if an invalid backup phrase is input.  Input should check if there are 24 words (or >= 12 if we are to support 128 bit phrases).

Should be just adding in a length condition like `phrase.trim().split(/\s+/g).length >= 12`.

### Current Behavior

Button is enabled if the input is not empty (e.g. a single letter).

### PR DESCRIPTION
**Pull request by Pedro-vk**
### Description

_Disable import wallet button when the backup phrase is not valid_

### Tested

_Changes were tested on Android emulator, going to import wallet view and filling the textarea with different words, checking that adding the 12th word (or first letter of the word) enables the button_

### Related issues

- Fixes #802 

### Backwards compatibility

_It doesn't affect compatibility_

### CODE REVIEW

File: packages/mobile/src/import/ImportWallet.tsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ export class ImportWallet extends React.Component<Props, State> {
           </View>
         )}
         <GethAwareButton
-          disabled={isSubmitting || !this.state.backupPhrase}
+          disabled={isSubmitting || this.state.backupPhrase.trim().split(/\s+/g).length < 12}
```
Comment (jmrossy) [inline packages/mobile/src/import/ImportWallet.tsx]: Consider creating an `isBackupPhraseValid` method and doing this check there

Comment (jmrossy) [reply packages/mobile/src/import/ImportWallet.tsx]: Also, did you mean 24 instead of 12?

**Review by jmrossy — CHANGES_REQUESTED** [review]
Nice first pass, see comments below.
For the Tested section of the PR, please provide specific details about how you tested the change.

File: packages/mobile/src/import/ImportWallet.tsx (hunk 1 above)
Comment (Pedro-vk) [reply packages/mobile/src/import/ImportWallet.tsx]: The issue considers both: 12 and 24. Is it ok?

Comment (Pedro-vk) [reply packages/mobile/src/import/ImportWallet.tsx]: > Consider creating an `isBackupPhraseValid` method and doing this check there

Sure!

File: packages/mobile/src/import/ImportWallet.tsx [hunk 2]
```diff
@@ -126,6 +126,10 @@ export class ImportWallet extends React.Component<Props, State> {
     }
   }
 
+  isBackupPhraseValid() {
```
Comment (jmrossy) [inline packages/mobile/src/import/ImportWallet.tsx:129]: Guess this should be `isBackupPhraseInvalid`

Comment (Pedro-vk) [reply packages/mobile/src/import/ImportWallet.tsx:129]: Fixed! And added more details on _tested_ section. Is it ok?

Comment (jmrossy) [reply packages/mobile/src/import/ImportWallet.tsx:129]: Is it fixed? Still shows as Valid for me

**Review by jmrossy — APPROVED** [review]

**Pull request merged**